polkadot/runtime/westend/src/genesis_config_presets.rs

```diff
@@ -202,7 +202,10 @@ fn westend_testnet_genesis(
 				.iter()
 				.map(|x| (x.0.clone(), x.0.clone(), STASH, StakerStatus::<AccountId>::Validator))
 				.collect::<Vec<_>>(),
-			invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect::<Vec<_>>(),
+			invulnerables: BoundedVec::try_from(
+				initial_authorities.iter().map(|x| x.0.clone()).collect::<Vec<_>>()
+			)
+			.expect("Too many invulnerable validators!"),
```

Comment:
Error message can be a bit more informative, hinting at which config should be tweaked if this `expect` is ever reached.

Reply:
error message improved in https://github.com/paritytech/polkadot-sdk/pull/6445/commits/0d47dc565fbb24c5e0e44f67b36e3ffff160af6b